index.js

```diff
@@ -2572,6 +2574,25 @@
   }
   S.not = def('not', {}, [$.Boolean, $.Boolean], not);
 
+  //# complement :: (a -> Boolean) -> a -> Boolean
+  //.
+  //. Takes a unary predicate and a value of any type, and returns the logical
+  //. negation of applying the predicate to the value.
+  //.
+  //. See also [`not`](#not).
+  //.
+  //. ```javascript
+  //. > Number.isInteger('foo')
+  //. false
+  //.
+  //. > S.complement(Number.isInteger, 'foo')
+  //. true
+  //. ```
```

Comment:
I like the choice of `Number.isInteger` as the predicate, but I'd like to pretend that its type is `Number -> Boolean` rather than `Any -> Boolean`. I suggest:

```javascript
> Number.isInteger(42)
true

> S.complement(Number.isInteger, 42)
false
```